hadoop-ozone/dist/src/main/smoketest/s3/MultipartUpload.robot

```diff
@@ -74,17 +70,16 @@ Test Multipart Upload Complete
                         Should contain          ${result}    UploadId
 
 #upload parts
-	${system} =         Evaluate    platform.system()    platform
-	Run Keyword if      '${system}' == 'Darwin'  Create Random file for mac
-	Run Keyword if      '${system}' == 'Linux'   Create Random file for linux
-	${result} =         Execute AWSS3APICli     upload-part --bucket ${BUCKET} --key multipartKey1 --part-number 1 --body /tmp/part1 --upload-id ${uploadID}
-	${eTag1} =          Execute and checkrc     echo '${result}' | jq -r '.ETag'   0
-	                    Should contain          ${result}    ETag
+    ${system} =         Evaluate    platform.system()    platform
+    Run Keyword         Create Random file      5
+    ${result} =         Execute AWSS3APICli     upload-part --bucket ${BUCKET} --key multipartKey1 --part-number 1 --body /tmp/part1 --upload-id ${uploadID}
+    ${eTag1} =          Execute and checkrc     echo '${result}' | jq -r '.ETag'   0
+                        Should contain          ${result}    ETag
 
                         Execute                 echo "Part2" > /tmp/part2
-	${result} =         Execute AWSS3APICli     upload-part --bucket ${BUCKET} --key multipartKey1 --part-number 2 --body /tmp/part2 --upload-id ${uploadID}
-	${eTag2} =          Execute and checkrc     echo '${result}' | jq -r '.ETag'   0
-	                    Should contain          ${result}    ETag
+    ${result} =         Execute AWSS3APICli     upload-part --bucket ${BUCKET} --key multipartKey1 --part-number 2 --body /tmp/part2 --upload-id ${uploadID}
+    ${eTag2} =          Execute and checkrc     echo '${result}' | jq -r '.ETag'   0
+                        Should contain          ${result}    ETag
```

Comment:
Are these unintended changes? As I see the change is indentation.

Reply:
Yetus asked me to not use tab. Without fixing the tabs the patch would cause a whitespace mismatch (my new lines with spaces and the old lines with tabs). So it's intentional, but I can move to a different